nakadi-scopes.txt

```diff
@@ -0,0 +1,7 @@
+Resource Type ID: nakadi_event
+Resource Type Name: Nakadi Event
+Owning team: aruha
+Application Scope: nakadi_event.read
```

Comment:
Yeah, true. But that will be another resource. Can be something like `nakadi_event_type` with scopes also `read` and `write`.